tools/clang/tools/dotnetc/EditorModels.cs

```diff
@@ -505,9 +505,7 @@ public SettingsManager()
         {
             this.doc = new XDocument(new XElement("settings"));
             this.SettingsPath =
-                System.IO.Path.Combine(
-                    System.Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData),
-                    "dndxc",
+                System.IO.Path.Combine(AppDomain.CurrentDomain.BaseDirectory,
                     "settings.xml");
```

Comment:
Shouldn't this be named after the tool, such as dndxc.settings.xml or something like that?